data/json/items/software.json

```diff
@@ -62,6 +62,24 @@
     "price": "2 USD",
     "price_postapoc": "0 cent"
   },
+  {
+    "id": "software_encryption_codes",
+    "copy-from": "software",
+    "type": "GENERIC",
+    "name": { "str_sp": "encryption codes" },
+    "description": "There are numerous security keys with dates for potential future access to encrypted archived data.",
+    "price": "2 USD",
+    "price_postapoc": "0 cent"
+  },
+  {
+    "id": "software_archived_data",
+    "copy-from": "software",
+    "type": "GENERIC",
+    "name": { "str_sp": "archive data" },
+    "description": "Extremely compressed research data, observation and experiment statistics.  The only pattern you manage to recognize is an increase in report amount closer to Cataclysm, but also a reduction in the number of facilities still submitting reports.",
```

Comment:
```suggestion
    "description": "Extremely compressed research data, observation and experiment statistics.  The only pattern you manage to recognize is an increase in the amount of report closer to the Cataclysm, but also a reduction in the number of facilities still submitting reports.",
```

Reply:
btw i m pretty sure this can't be read in game or at least i don't know how to read description of software in game